src/scss/components/_navigation.scss

```diff
@@ -0,0 +1,52 @@
+.header {
+  width: 100%;
+}
+
+.navigation {
+  @include position-fixed;
+  background-color: $c-pure-white;
+
+  &__container {
+    @include flex-row-space-between;
+    width: 90%;
+    margin: 0 auto;
+    padding: 2.5rem 0;
+    max-width: 123.75rem;
+  }
+
+  &__logo {
+    margin: 0 auto;
+    width: 12.5rem;
+    @media (min-width: $bp-sm) {
+      margin: 0 auto;
+      width: 15.5rem;
+    }
+
+    @media (min-width: 990px) {
+      margin: 0;
+      width: 16.75rem;
+    }
+  }
+
+  &__list {
+    display: none;
+    @include default-link-ul;
+
+    @media (min-width: 990px) {
```

Comment:
By any chance, do you have a bp-variable for this?

Reply:
You're right. I'll make one now